docs/timeseries/io.rst

```diff
@@ -0,0 +1,94 @@
+.. _timeseries-io:
+
+Reading/writing time series
+***************************
+
+.. |Table| replace:: :class:`~astropy.table.Table`
+.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
+.. |BinnedTimeSeries| replace:: :class:`~astropy.timeseries.BinnedTimeSeries`
+
+Build-in readers
+================
+
+Since |TimeSeries| and |BinnedTimeSeries| are sub-classes of |Table|,
+they have :meth:`~astropy.table.Table.read` and
+:meth:`~astropy.table.Table.write` methods that can be used to read time series
+from files. We include a few readers for well-defined formats in `astropy.timeseries` -
+for example we have readers for light curves in FITS format from the
+`Kepler <https://www.nasa.gov/mission_pages/timeseries/main/index.html>`_ and
+`TESS <https://tess.gsfc.nasa.gov/>`_ missions.
+
+Here is an example of using Kepler FITS time series - we start off by fetching an example
+file:
+
+.. plot::
+   :include-source:
+   :context: reset
+   :nofigs:
+
+   from astropy.utils.data import get_pkg_data_filename
+   example_data = get_pkg_data_filename('timeseries/kplr010666592-2009131110544_slc.fits')
+
```

Comment:
This would go with the identical suggestion I made in the getting started, presumably.

```suggestion

.. note::
    The light curve provided here is hand-picked for example purposes.  To get
    other Kepler light curves for science purposes using Python, see the
    `astroquery <https://astroquery.readthedocs.io>`_ affiliated package.
    
```